modules/frontend/config.go

```diff
@@ -38,6 +38,8 @@ type SearchConfig struct {
 	Timeout time.Duration       `yaml:"timeout,omitempty"`
 	Sharder SearchSharderConfig `yaml:",inline"`
 	SLO     SLOConfig           `yaml:",inline"`
+	// TODO: can we reuse the search config? do we need another set of config here?? decide
+	MetadataSLO SLOConfig `yaml:"metadata_slo,omitempty"`
```

Comment:
This seems fine to me.  The request can be within SLO as long as it is under the duration or over the throughput.   Are there other considerations?